services/TikTok.json

```diff
@@ -14,6 +14,29 @@
         "startBefore": "#privacy-eea",
         "endBefore": "#privacy-row"
       }
+    },
+    "communityGuidelines": {
+      "fetch": "https://www.tiktok.com/community-guidelines?lang=en",
+      "select": "article"
```

Comment:
Use `.base-layout-container` for `select` to also select the document title "Community Guidelines"